packages/core/helper-plugin/src/content-manager/CMEditViewDataManager.tsx

```diff
@@ -0,0 +1,102 @@
+import * as React from 'react';
+
+import { MessageDescriptor } from 'react-intl';
```

Comment:
You need this as opposed to the TranslationMessage in the types file?

Reply:
Looking at how `formErrors` was used, it looked like it was only ever used with the `MessageDescriptor` shape. I think we can change it when working more in the CM if it doesn't fit in reality?